submissions/Oleksandr-Heleta/task_js_oop/app.js

```diff
@@ -0,0 +1,131 @@
+const START_X = 202,
+    START_Y = 404,
```

Comment:
As a peer developer I am tasked to change the playfield params. Can these pixel values be calculated from such abstractions as row and column so that I do not need to calculate them myself?